test/src/examples_complete_test.go

```diff
@@ -23,10 +23,4 @@ func TestExamplesComplete(t *testing.T) {
 
 	// This will run `terraform init` and `terraform apply` and fail the test if there are any errors
 	terraform.InitAndApply(t, terraformOptions)
-
-	// Run `terraform output` to get the value of an output variable
-	outputExample := terraform.Output(t, terraformOptions, "example")
-
-	// Verify we're getting back the outputs we expect
-	assert.Regexp(t, "^Hello world! [0-9]+$", outputExample)
```

Comment:
The arn will contain the label, no?

Reply:
https://github.com/cloudposse/terraform-aws-ssm-parameter-store/blob/39ede1cdc07e8c46c82e778e45db2b7489948449/test/src/examples_complete_test.go#L32

Reply:
I think better to have one assertion at least. Assert that the label was used by checking the ARN. 

Since we removed all asserts, it's nagging about the orphaned import.
![image](https://user-images.githubusercontent.com/52489/89048915-50ac7c80-d305-11ea-90f6-d4bc35ecf1ba.png)

Reply:
Yep, just today learned how make our `Makefile` for tests work.
We have `GOEXE   ?= /usr/bin/go`, and such path on MacOSX does not exist. I don't think you can move/cp/link binaries there so such variable should be overwritten. 

**Tests fixed**